CMakeLists.txt

```diff
@@ -210,11 +218,29 @@ function(list_licenses OUTPUT MODULES)
     file(APPEND ${OUTPUT} ")FILAMENT__\"\n")
 endfunction(list_licenses)
 
+function(include_thirdparty NAME)
```

Comment:
I'm really not fond of this, we should understand why we have warnings. Suppressing them all blindly is bad, we may miss real problems when we update to newer versions of third party libraries.

Reply:
Hm you're right, and this mostly relates to upstream projects not considering clang-cl.exe in their build projects. Getting all necessary patches to remove those issues accepted upstream will take a while though. I might try a different approach... (i.e. -Wno-unknown-argument to suppress compiler flags unknown to clang-cl.exe).

Reply:
Not that except for `spirv-tools` we have our own `CMakeLists.txt` for each project in `third_party`. They are in a `tnt/` sub-directory of every library (TNT was an early codename for our project). This is where we use library specific flags (and we already turn off specific warnings). No need to bother trying to upstream all the changes. I just don't want to blindly turn everything off :)